src/sidebar/store/modules/test/groups-test.js

```diff
@@ -7,11 +7,42 @@ describe('sidebar.store.modules.groups', () => {
   const publicGroup = {
     id: '__world__',
     name: 'Public',
+    isMember: true,
+    isScopedToUri: true,
   };
 
   const privateGroup = {
-    id: 'foo',
+    id: 'privateid',
     name: 'Private',
+    isMember: true,
+    isScopedToUri: true,
+  };
+
+  const restrictedGroup = {
+    id: 'restrictid',
+    name: 'Restricted',
+    isMember: false,
+    isScopedToUri: true,
+  };
+
+  const restrictedUnscopedGroup = {
```

Comment:
Is it that the group is _unscoped_ OR that the group is currently not within its scope?